bcipy/signal/model/evaluate/fusion.py

```diff
@@ -0,0 +1,370 @@
+import numpy as np
+from sklearn.utils import resample
+
+from bcipy.config import (TRIGGER_FILENAME)
+from bcipy.helpers.acquisition import analysis_channels
+from bcipy.helpers.stimuli import update_inquiry_timing
+from bcipy.helpers.symbols import alphabet
+from bcipy.helpers.triggers import TriggerType, trigger_decoder
+from bcipy.preferences import preferences
+from bcipy.signal.model.base_model import SignalModelMetadata
+from bcipy.signal.model.gaussian_mixture import (GaussianProcess)
+from bcipy.signal.model.pca_rda_kde import PcaRdaKdeModel
+from bcipy.signal.process import (ERPTransformParams, extract_eye_info,
+                                  filter_inquiries, get_default_transform)
+
+
+def calculate_eeg_gaze_fusion_acc(
+        eeg_data,
+        gaze_data,
+        device_spec_eeg,
+        device_spec_gaze,
+        symbol_set,
+        parameters,
+        data_folder):
```

Comment:
Return statement `-> Tuple[list, list, list]`? We may want the average or max acc returned. `-> Tuple[str, str, str]`

Reply:
Does `Tuple[float, float, float]` work?